### ISSUE TITLE
Visualizing annotations within files

### ISSUE THREAD

**Issue by danyaljj** [CogComp/apelles#35]
Suppose I want to visualize annotations within json (annotation) files. 
Is there a way to do it? (note that this is not diff-ing, but it should be easier). 

@mssammon @lxtbell 

@ChaseDuncan this was your question; right?

**Comment by ChaseDuncan** [comment]
Yes. In particular, I'm interested in viewing annotations which aren't necessarily created by a system in the pipeline. An example file can be found at:

/shared/experiments/cddunca2/coherence-examples-el/michael_jordan.json

I'd like to be able to visualize the English_WIKIFIERVIEW and NGD views.

**Comment by mssammon** [comment]
You can just specify a single directory argument instead of multiple directories.  If the name is not already in the available view list, in `apelles/public/comparison.html` edit the selectpicker with id="your_view_name"
